pkg/volume/rbd/rbd_util.go

```diff
@@ -230,6 +231,21 @@ func execRbdMap(b rbdMounter, rbdCmd string, mon string) ([]byte, error) {
 	}
 }
 
+// check if rbd-nbd tools are installed
+func checkRbdNbdTools(e mount.Exec) bool {
```

Comment:
Should we create a checkRbdKrbdTools to get the similar logic (e.g. modprobe krbd) out of other places?

Reply:
I don't think so. There is only one modprobe for krbd and there is no bailing even if it fails.